env/env.go

```diff
@@ -62,3 +64,21 @@ const (
 
 // LookupFunc defines a function to look up a key from the environment.
 type LookupFunc func(key string) (string, bool)
+
+// EmptyLookup is a LookupFunc that always returns "" and false.
+func EmptyLookup(key string) (string, bool) { return "", false }
+
+// Lookup is a LookupFunc that uses os.LookupEnv.
+func Lookup(key string) (string, bool) { return os.LookupEnv(key) }
```

Comment:
I wonder if this should be called `OSLookup`?

Reply:
I thought about it too, but I wanted to keep it generic and added documentation instead, saying that it uses `os.LookupEnv`.